ColabTool/LinkPost/index.jsx

```diff
@@ -0,0 +1,128 @@
+import React, { PropTypes } from 'react';
+import {
+  Card,
+  Image,
+  LinkifiedText,
+  Text
+} from '@bufferapp/components';
+import style from './style.css';
+import Post from '../Shared/Post';
+
+// TODO: Padding between inner/outer card 0.5 for left, right, bottom?
+const LinkPost = ({
+  isConfirmingDelete,
+  isDeleting,
+  isWorking,
+  hovered,
+  imageSrc,
+  links,
+  linkAttachment,
+  manager,
+  onApproveClick,
+  onCancelConfirmClick,
+  onDeleteClick,
+  onDeleteConfirmClick,
+  onEditClick,
+  onMouseEnter,
+  onMouseLeave,
+  profile,
+  text,
+}) => {
+  const children = (
+    <div className={style['post-content']}>
+      <div className={style['post-content-text']}>
+        <LinkifiedText
+          links={links}
+        >
+          {text}
+        </LinkifiedText>
+      </div>
+      <div className={style['link-attachment-container']}>
+        <Card
+          hovered={hovered}
+          onMouseEnter={onMouseEnter}
+          onMouseLeave={onMouseLeave}
+          noPadding
+        >
+          <div className={style['link-attachment-content']}>
+            <div className={style['link-attachment-image']}>
+              <Image
+                src={imageSrc}
+                width={'15em'}
+                height={'100%'}
+                maxWidth={'15em'}
+                maxHeight={'15em'}
+                border={'rounded'}
+              />
+            </div>
+            <div className={style['link-attachment-text']}>
+              <span className={style['link-title']}>
```

Comment:
@hharnisc - ah good one! Hm, I just thought using order with flexbox was super clear about where we want each thing to be. Definitely not married to either approach - it feels like using div's (like in your example above) might be a bit cleaner and easier to read. Feels good to give that a go! 😄